### PR TITLE
Improve Conversion API overview

### PR DESCRIPTION
**Pull request by marklewin**
## Description

Addresses [DEVX-1098](https://nexmoinc.atlassian.net/browse/DEVX-1098)

### CODE REVIEW

**Comment by marklewin** [comment]
Marking as don't merge so that I can get the PM and others to look at this first.

**Comment by mheap** [comment]
@marklewin If we don't get a reply this week let's merge

**Comment by marklewin** [comment]
Agreed - chased Drosos again yesterday and although he said he'd look at it
he hasn't come back to me.

**Comment by marklewin** [comment]
Drosos, the PM for the Conversion API has approved the rewrite so this is ready for merge

File: _documentation/messaging/conversion-api/overview.md [hunk 1]
```diff
@@ -5,45 +5,42 @@ meta_title: Conversion API
...
 
-The conversion data you send us is confidential: Nexmo does not share it with third parties.
+> **Note**: If you are using the [Verify API](/verify/overview) for 2FA, you don't need to send us conversion data. The Verify API does this automatically.
 
-In order to identify the carriers who provide the best performance, Nexmo continually tests the routes we use to deliver SMS and voice calls. Using Adaptive Routing™, Nexmo actively reroutes messages through different carrier routes and ensures faster and more reliable delivery for your messages. The route choice is made using millions of real-time conversion data points.
+Nexmo uses this conversion data, together with [delivery receipts (DLRs)]() to facilitate Adaptive Routing™. The Adaptive Routing algorithm automatically determines the best carrier routes via which to deliver SMS and voice calls at any specific moment. In most instances, a DLR is confirmation that your message was delivered to the recipient. However, not all carriers' DLRs are reliable and some do not provide them at all. Therefore, conversion data is our best indicator of route quality.
```
Comment (tbedford) [inline _documentation/messaging/conversion-api/overview.md]: Missing link? `[delivery receipts (DLRs)]()`

File: _documentation/messaging/conversion-api/overview.md [hunk 2]
```diff
@@ -5,45 +5,42 @@ meta_title: Conversion API
...
 
-Message delivery indicators are:
+> **The conversion data you send us is confidential: Nexmo will never share it with third parties.**
```
Comment (tbedford) [inline _documentation/messaging/conversion-api/overview.md]: We would normally probably be consistent on callout format on this page. i.e. Not bold, but use Note, or Important.

Comment (marklewin) [reply _documentation/messaging/conversion-api/overview.md]: Good call, changed to **Note**

File: _documentation/messaging/conversion-api/overview.md (hunk 1 above)
Comment (marklewin) [reply _documentation/messaging/conversion-api/overview.md]: Fixed

**Pull request merged**